pkgs/by-name/co/cosmic-wallpapers/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "cosmic-wallpapers";
+  version = "1.0.0-alpha.2";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = "cosmic-wallpapers";
+    rev = "refs/tags/epoch-${finalAttrs.version}";
+    hash = "sha256-t98ZDmazsPxXNXI4cNDkIo61/eKQaJg9lZa4CsfX00g=";
+  };
+
+  makeFlags = [ "prefix=${placeholder "out"}" ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
You're going to want `extraArgs = [ "--version-regex" "epoch-(.*)" ];` if you want nix-update to not immediately heck up the version

Also once again, if you are going to copy/paste from <https://github.com/lilyinstarlight/nixos-cosmic> that is alright, but please follow the license terms (which would include attribution that must be distributed with this file)

Reply:
Ah, I thought you meant cosmic-wallpaper's license terms. I'll mark you as a co-author in git, which should be sufficient.

Reply:
MIT license requires attribution with distribution. The "co-author" fields in git commits are not included in nixpkgs tarballs and insufficient

Reply:
You could also just rewrite it. Also heads-up I finished the derivation yesterday, including having it use the git-lfs fetcher. So without that this derivation has LFS pointer text files instead of some images

Reply:
I'll just rewrite it then.

Reply:
Rewrite complete. None of your copyrighted code is used in my derivation, as I rewrote it from the ground up.

Reply:
Nice work tracking down licenses too :)

I'd spot checked several, but it appears trusting what sources said was a mistake